src/chains/ethereum/block/src/block.ts

```diff
@@ -93,7 +93,7 @@ export class Block {
       // get the maxFeePerGas and maxPriorityFeePerGas, use those to calculate
       // the effectiveGasPrice and add it to `extra` above, or we can just
       // leave it out of extra and update the effectiveGasPrice after like this
-      tx.updateEffectiveGasPrice(header.baseFeePerGas);
+      tx.updateEffectiveGasPrice(header.baseFeePerGas?.toBigInt());
```

Comment:
What this an existing bug?

Reply:
No, this was introduced when I changed the `updateEffectiveGasPrice` function to receive a bigint rather than a quantity. For transactions types that are pre-1559, the `updateEffectiveGasPrice` function is a no-op (the `effectiveGasPrice` is just the static `gasPrice` param passed by the user), so the `.toBigInt()` was never called within the function for those. If we called the function passing an undefined `Quantity`, it didn't matter. Now, we're converting on the outside so we have to prevent calling `.toBigInt` on an undefined base fee.